addons/website_event/views/event_event_views.xml

```diff
@@ -9,7 +9,19 @@
         <field name="arch" type="xml">
             <xpath expr="//group[@name='right_event_details']/field[@name='company_id']" position="after">
                 <field name="website_id" invisible="1"/>
+                <field name="website_published" invisible="1"/>
                 <field name="website_id" options="{'no_create': True}" domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]" groups="website.group_multi_website"/>
+                <label for="visibility" string="Visibility"/>
+                <div class="d-flex">
+                    <div class="col-5">
```

Comment:
Strange to play with cols in a form, without specifying row on top and without having another div to complete the remainder of the 12 columns.